interface/VideoEncoderDefs.h

```diff
@@ -152,6 +153,11 @@ typedef struct VideoFrameRate {
     uint32_t frameRateDenom;
 }VideoFrameRate;
 
+typedef struct SVCTVideoFrameRate {
+    VideoFrameRate fraction[32];
+    uint8_t num;
```

Comment:
Here is the interface to user, would better add some comment to describe what does fraction and num mean.